src/components/404/NotFoundError.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+class NotFoundError extends React.Component {
+  render() {
```

Comment:
this is a statelless component please change this. if you have eslint set up properly you would see the error